docs/guides/insar_product_guide.md

```diff
@@ -178,18 +218,20 @@ As with our RTC products, HyP3 product names are packed with information pertain
 
 All of the main InSAR product files are 32-bit floating-point single-band GeoTIFFs.
 
-- The amplitude image is the amplitude from the reference scene. 
-- The coherence file contains values from 0.0 to 1.0, with zero being non-coherent to 1.0 being perfectly coherent. 
-- The unwrapped phase file shows the results of the phase unwrapping process.  This is the main interferogram output. 
-- The line-of-sight displacement file contains the displacement directly away from the satellite.
-- The vertical displacement is just the vertical component of that displacement. 
-- The looks vectors, &#966 and &#952 describe the look vector elevation and orientation angles. 
-- The DEM file gives the local terrain heights
-- The water mask image has zeros for water and 1 for land
-- The incidence angle map gives the local incidence angle of the terrain
+- The amplitude image is the calibrated radiometric backscatter from the reference granule in sigma-nought power. The image is terrain corrected using a geometric correction, but not radiometrically corrected. 
+- The coherence file contains values from 0.0 to 1.0, with 0.0 being completely non-coherent and 1.0 being perfectly coherent. 
+- The unwrapped phase file shows the results of the phase unwrapping process. This is the main interferogram output. 
+- The line-of-sight displacement file indicates the displacement in meters along the look direction of the sensor (perpendicular to the direction of movement of the satellite).
+- The vertical displacement is generated from the line of sight displacement values and the DEM, and makes assumptions that likely do not hold true for most deformation processes that are impacted by horizontal motion as well as vertical motion.
```

Comment:
@talogan I've got some vagueness about the vertical displacement map generation, and could benefit from a chat about it. Some of the references may need to be clarified/adjusted.

Reply:
This is rather alarmist, isn't it?  Why would you say 'makes assumptions that likely do not hold true for most deformation?

Reply:
Based on what Franz teaches in his class, there's no way to extract vertical vs. horizontal components reliably from a single interferogram. You'd either have to have a series of interferograms or have GPS data to be able to tease out their relative contributions from the line-of-sight movement.

Reply:
Amended in later [commit](https://github.com/ASFHyP3/hyp3-docs/pull/96/commits/3b9132e4d3f42cce4d4a8f73182448c047f85a01)